internal/index/manager/manager.go

```diff
@@ -456,8 +469,13 @@ func (t tag) converterNames() []string {
 }
 
 func (mgr *Manager) Close() {
-	if mgr.watcher != nil {
-		if err := mgr.watcher.Close(); err != nil {
+	if mgr.convertersWatcher != nil {
+		if err := mgr.convertersWatcher.Close(); err != nil {
+			log.Printf("Failed to close watcher: %v", err)
```

Comment:
Can you add the watcher type to the error messages here too to allow to disambiguate which watched folder was mishaving please?